.gitmodules

```diff
@@ -22,4 +22,7 @@
 	path = WorldsCollide_location_gating1
 	url = https://github.com/BriGuy7727/WorldsCollide.git
 	branch = loc-gated
-
+[submodule "WorldsCollide_shuffle_by_world"]
+	path = WorldsCollide_shuffle_by_world
+	url = https://github.com/JackQuincy/WorldsCollide.git
+	branch = chest-shop-suffle-by-world
```

Comment:
I might be changing this branch to one that has these changes and BriGuy7727's changes for location gating.